client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -151,10 +151,8 @@ const availableFlows: Record< string, () => Promise< { default: Flow } > > = {
 		import(
 			/* webpackChunkName: "site-migration-flow" */ '../declarative-flow/site-migration-flow'
 		),
-	[ HOSTED_SITE_MIGRATION_V2_FLOW ]: () =>
-		import(
-			/* webpackChunkName: "hosted-site-migration-flow-v2" */ '../declarative-flow/hosted-site-migration-v2'
-		),
+	[ MIGRATION_FLOW ]: () =>
```

Comment:
Should we update it to use the https://github.com/Automattic/wp-calypso/blob/7749853c5bfa4d8df361547d06fb95fde67273d3/client/landing/stepper/declarative-flow/registered-flows.ts#L37-L38 IMPORT_FOCUSED const?

Reply:
Good catch!

https://github.com/Automattic/wp-calypso/pull/93388/commits/716456c15b8503c716089f51bf9b27fe57b53cf4